### PR TITLE
Add release sequence to release notes for latest version

### PR DESCRIPTION
**Pull request by jalexander-fastly**
Add release sequence for latest TFE release

### CODE REVIEW

File: v202111-1.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# TFE Release v202111-1
+# TFE Release v202111-1 (581)
```
Comment (aaron-lane) [inline v202111-1.md]: v202111-1 is actually 582, not 581:

```suggestion
# TFE Release v202111-1 (582)
```

**Review by aaron-lane — CHANGES_REQUESTED** [review]
Thanks for catching this @jalexander-fastly 🎉 

I have one requested change, and please also sign the CLA.

File: v202111-1.md (hunk 1 above)
Comment (jalexander-fastly) [reply v202111-1.md]: @aaron-lane I was going off of this doc. 😄  https://support.hashicorp.com/hc/en-us/articles/1500009817722-Terraform-Enterprise-Release-Sequence-Versions.

I will change to 582.

Thanks!

Comment (aaron-lane) [reply v202111-1.md]: Thanks for pointing that out! I'll make sure that document is corrected as well 😅

**Review by aaron-lane — APPROVED** [review]
LGTM 🧶

**Pull request merged**